src/pages/dar_application/ResearcherInfo_new.js

```diff
@@ -182,7 +181,6 @@ export default function ResearcherInfo(props) {
         ]),
 
         div({className: 'dar-application-row'}, [
-          // TODO: DUOS-1754
           h3('1.5 Internal Collaborators'),
           div(
             `Please add Internal Collaborators here Internal Collaborators are defined as individuals
```

Comment:
```suggestion
            `Please add Internal Collaborators here. Internal Collaborators are defined as individuals
```

Reply:
Just a suggestion to bring the text in line with the External Collaborators text.